Cake-css.css

```diff
@@ -0,0 +1,123 @@
+.hamIcon{
+    position: absolute;
+    right: 10px;
+    top: 20px;
+}
+
+.logo_image{
+    display: flex;
+    justify-content: center;
+    height: 200px;
+    background: url("images/Mynavbarimage.jpeg");
+    background-position: center;
+}
+.nav_bar{
+    background: url("images/nav_back.JPG");
+   
+}
+
+.mainMenu{
+    font-family: 'Courgette', cursive;
+    list-style: circle;
+    right:2px;
+    top: 100px;
+    padding: 30px;
+    font-size: 30px;
+    height: 200px;
+    border-radius:10px;
+    border:4px double plum;
+    text-decoration: none;
+
+}
+body{
+    background: url("images/wallpaper.jpg");
+    text-decoration: none;
+}
+
+a:hover{
+    color:blue;
+    text-decoration: none;
+}
+.H-one{
+    color:#fc0384;
+    font-size: 50px;
+    font-family: 'Courgette', cursive;
+    /*display: flex;
+    flex-direction: column;
+    justify-content:center;*/
+    text-align: center;
+    padding: 50px 20px 10px 50px;
+
+}
+.description{
+    font-size: 30px;
+    text-align: center;
+    color: black;
+    font-family: 'Courgette', cursive;
+    padding: 0 auto;
+    margin: 10 10;
+}
+.main_cakes{
+    display:flex;
+    justify-content:space-around;
+    align-content:space-around;
+    flex-wrap: wrap;
+}
+.cakes{
+    display: block;
+    box-sizing: border-box;
+}
+.style_cake{
+    height: 300px;
+    padding: 100px 100px;
```

Comment:
This large padding is interfering with the layout of the page on mobile (the elements are wider than the size of the screen). If you remove the padding and add some extra styling to the `cakes` class (hint: use flexbox), it should fix this.